contracts/DonationCampaigns.sol

```diff
@@ -42,13 +161,25 @@ contract DonationCampaigns {
 		);
 	}
 
+    /** @dev Create a new campaign.
+      * @param _name The name of the campaign.
+      * @param _image The image associated with the campaign.
+      * @param _description The description of the campaign.
+      * @param _goal The fundraising goal of the campaign.
+      * @param _campaignClosedDate The date when the campaign will be closed.
+    */
     function createCampaign(
 		string memory _name,
 		string memory _image,
 		string memory _description, 
 		uint _goal,
         uint _campaignClosedDate
-	) public {
+	) public onlyOwner {
```

Comment:
This is not needed, the idea is anybody can create campaign to collect donation, not only contract owner